tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = pep8,py27,py36
```

Comment:
Just curious... why PEP-8 which is basically a style guide should be listed here alongside actual python versions?...

Reply:
`envlist` contains list of default virtual environments which should be created when command `tox` will be executed without selecting envs by argument `-e`, like `tox -e venv`.

`venv` is defined in `tox.ini` also, but it is not intended to be used in automated tests.

More examples of `tox.ini`:

- https://tox.readthedocs.io/en/latest/example/basic.html#a-simple-tox-ini-default-environments
- https://tox.readthedocs.io/en/latest/examples.html

I used those tox'es:

- https://git.openstack.org/cgit/openstack-infra/git-review/tree/tox.ini
- https://github.com/pasuder/labs/blob/master/tox.ini